src/pages/Search/Suggestions.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function Suggestions() {
+  return (
+    <div>
+      <Title>밀리 추천 검색어</Title>
+      <KeywordSearch>
+        <IconBox iconSize="1rem">
+          <i className="fas fa-search" />
+        </IconBox>
+        <span>밀리와 영어공부, 지금 시작해요!</span>
+      </KeywordSearch>
+      <KeywordSearch>
+        <IconBox iconSize="1rem">
+          <i className="fas fa-search" />
+        </IconBox>
+        <span>가을 감성 듬뿍, 로맨스 베스트</span>
+      </KeywordSearch>
+    </div>
+  );
+}
+
+const KeywordSearch = styled.div`
+  margin-left: 10px;
+  color: ${props => props.theme.middleGrey};
+  font-family: 'Noto Sans KR', sans-serif;
+`;
+
+const IconBox = styled.span`
+  cursor: pointer;
+  margin: 5px;
+  font-size: ${props => props.iconSize};
```

Comment:
```suggestion
  font-size: ${({ iconSize }) => iconSize ? iconSize : '1rem'};
```
별도의 props가 주어지면 그 값을 부여하고 없을 때는 1rem으로 기본값을 주는 건 어떨까요?

Reply:
수정 완료했습니다

Reply:
@janine-kang 수정 안된거 같습니다! 아까 같이 봤던 내용처럼 이 부분도 수정해줄 수 있을지 한번 살펴봐주세요